### ISSUE TITLE
Change color

### ISSUE THREAD

**Issue by kathcode** [disq/HighlightWhitespaces#17]
I need to change the highlight color, but don´t understand how you should do. I try two ways.
{
    "highlight_whitespaces_space_highlight_scope_name": "#dedede",
    "highlight_whitespaces_tab_highlight_scope_name": "#dedede",
    "highlight_whitespaces_eol_highlight_scope_name": "#dedede",
    "highlight_whitespaces_mixed_highlight_scope_name": "#dedede"
}

{
    "highlight_whitespaces_space_highlight_scope_name": "comment",
    "highlight_whitespaces_tab_highlight_scope_name": "comment",
    "highlight_whitespaces_eol_highlight_scope_name": "comment",
    "highlight_whitespaces_mixed_highlight_scope_name": "comment"
}

please help me, thanks

**Comment by nimylian** [comment]
@LeslieMeneses If you open the package using PackageResourceViewer(also a Sublime Package) you can open highlight_whitespace.py within it. If you change line 34, to "comment" from "invalid" it will change. You can also make it empty and turn the white spaces white. Unfortunately, I have no idea how to customize it beyond that.

```
DEFAULT_COLOR_SCOPE_NAME = "comment"
      or
DEFAULT_COLOR_SCOPE_NAME = ""
```

**Comment by diegomarquez** [comment]
Just figured out how to do this, it's a little convoluted...

Using **PackageResourceViewer** find the theme file you are using. I am using one of the default themes, Monokai.tmTheme, found in "Color Scheme - Default".

Open the file, scroll all the way to the bottom and insert the following piece of XML

```
<dict>
  <key>name</key>
  <string>Custom highlight color</string> <!-- Set the name you want here -->
  <key>scope</key>
  <string>custom.highlight.color</string> <!-- Set the id you want here -->
  <key>settings</key>
  <dict>
    <key>background</key>
    <string>#F93232</string> <!-- color goes here -->
    <key>fontStyle</key>
    <string></string>
    <key>foreground</key>
    <string>#F9F2CE</string> <!-- color goes here -->
  </dict>
</dict>
```

Save the changes.

_Note: Found that snippet in the **[TrailingSpaces](https://github.com/SublimeText/TrailingSpaces)** Package documentation, the package this one is forked from._

With that set, open the User Settings file of **HighlightWhitespaces** and add the following lines to it

```
{
   "highlight_whitespaces_space_highlight_scope_name": "custom.highlight.color",
   "highlight_whitespaces_tab_highlight_scope_name": "custom.highlight.color"
}
```

That should do it. You can even have different colors for spaces and tabs. Pretty cool. The only annoying part is that if you change color schemes you need to manually add the custom color to the new theme, which is a bit of a drag.

**Comment by abhiaj** [comment]
Adding upon the previous comment by @diegomarquez,

If you are using a custom theme like me, check which file you are using in Preferences->Color Scheme 
You will see something like this:
![Screenshot from 2020-05-14 17-49-02](https://user-images.githubusercontent.com/34074072/81933890-abd6a900-960b-11ea-8dd0-de6cbbe5e8bf.png)
 
As you can see, my current color scheme is Monokai- Contrast which is under a plugin colorsublime-themes.

Use packageresource viewer to go to color sublime and then select the monokai-contrast .tmtheme file

![Screenshot from 2020-05-14 17-54-44](https://user-images.githubusercontent.com/34074072/81934247-46cf8300-960c-11ea-8fbe-ee3fac53ea00.png)

![Screenshot from 2020-05-14 17-54-55](https://user-images.githubusercontent.com/34074072/81934253-4afba080-960c-11ea-963b-4b6391845d79.png)

This will now open a .tm theme file which is an xml file for defining your "scopes". "Scopes" can be thought of as color-schemas for now.

Now here is a catch, if you just paste the code at the bottom, it is not going to work. It needs to be inside the settings bracket of Monokai contrast theme.

`<dict>
	<key>name</key>
	<string>Monokai Contrast</string>
	<key>settings</key>
	<array>
		<dict>
                        <key>name</key>
			<string>Custom1</string>
			<key>scope</key>
			<string>custom1</string>
			<key>settings</key>
			<dict>
				<key>foreground</key>
				<string>#c8f0c540</string>
				<key>fontStyle</key>
				<string>italic</string>
			</dict>
		</dict>`

(closing brackets are not shown)
So basically you have to put it inside Monokai contrast's settings in my case. It will be same for most, if not all.

**WARNING**: Do NOT create another block of monokai (or whatever theme) settings in the .tmtheme file. You have to put it inside the settings and not create another settings. Attaching the .tmtheme file for reference. 

Note that you can add opacity values for a color. Which I prefer very much just by simply adding two digits after the color hex code (eg: #c8f0c540)

[sample.txt](https://github.com/disq/HighlightWhitespaces/files/4628384/sample.txt)
